msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1032,8 +1032,33 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
-|`actions.supportsNoItemContext`| Boolean | | | Allows task pane add-ins to activate without the Reading Pane enabled or a message selected.<br>Default value: `false`|
+|`actions.multiselect`| Boolean | | | Specifies whether the end user can select multiple items, such as multiple email messages, and apply the action to all of them.|
+|`actions.supportsNoItemContext`| Boolean | | | Allows task pane add-ins to activate without the Reading Pane enabled or a message selected. |
+|`customFunctions`| Object | | | Custom functions enable developers to add new functions to Excel by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as **SUM**. |
+|`customFunctions.functions`| Array | 20,000 | ✔️ | An array of custom function objects that define custom functions metadata. |
+|`customFunctions.functions.id`| String | 64 characters | ✔️ | Specifies a unique ID for the function. This ID can only contain alphanumeric characters and periods and shouldn't be changed after it's set. It must start with a letter and have a minimum of three characters. Use uppercase letters as a best practice. |
+|`customFunctions.functions.name`| String | 64 characters | ✔️ | The name of the function that end users see in Excel. In Excel, this function name is prefixed by the custom functions namespace that is specified in the manifest file. It's unique within the scope of the file and can only contain alphanumeric characters. It must start with a letter and have a minimum of three characters. It shouldn't be the same as any cells between **A1** and **XFD1048576**, any cells between **R1C1** and **R1048576C16384**, or any Excel 4.0 macro function (such as **RUN** or **ECHO**).Use uppercase letters as a best practice. |
+|`customFunctions.functions.description`| String | 128 characters | | Specifies the description of the function that end users see in Excel. |
+|`customFunctions.functions.helpUrl`| URL | 2,048 characters | | Specifies a URL that provides information about the function. It's displayed in a task pane. For example, `http://contoso.com/help/convertcelsiustofahrenheit.html.` |
+|`customFunctions.functions.parameters`| Array | 128 | ✔️ | Defines the input parameters for the function. |
+|`customFunctions.functions.parameters.name`| String | 64 characters | ✔️ | Specifies the name of the parameter. This name is displayed in Excel's IntelliSense. |
+|`customFunctions.functions.parameters.description`| String | 128 characters | | Specifies a description of the parameter that is displayed in Excel's IntelliSense. |
+|`customFunctions.functions.parameters.type`| String enum | | | Specifies the data type of the parameter. Can be `boolean`, `number`, `string`, or `any`, which allows you to use of any of the previous three types. If this property isn't specified, the data type defaults to `any`.|
```

Comment:
Hi @alison-mk,
A specific format is followed in Manifest article when explaining the accepted values and default value. For example, 
Supported values: `boolean`, `number`, `string`, or `any`
Default value: `any`
Kindly refer the article,https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema-dev-preview, if required and rephrase the descriptions.

Reply:
Thanks! I'll submit this update.